linkerd/app/outbound/src/switch_logical.rs

```diff
@@ -52,14 +53,28 @@ impl<S> Outbound<S> {
                                 tracing::debug!("Profile describes a logical service");
                                 return Ok(svc::Either::B(Logical::new(logical_addr, rx)));
                             }
+
+                            // If there was a profile but it didn't include an
+                            // endpoint or logical address, create a bare
+                            // endpoint from the original destination address,
+                            // using the profile-provided opaqeuness. This
```

Comment:
super unimportant nit, take it or leave it: this sentence is kind of long, i might break it up
```suggestion
                            // There was a profile but it didn't include an
                            // endpoint or logical address. Create a bare
                            // endpoint from the original destination address,
                            // using the profile-provided opaqueness. This
```